data/tooltips.txt

```diff
@@ -134,15 +134,27 @@ tip "hull:"
 tip "hull energy:"
 	`Energy consumed per second when repairing the hull.`
 
+tip "hull energy multiplier:"
+	`A multiplier that affects hull energy. A multiplier of 10% will increase the stat by 10%. Multipliers from multiple outfits are additive.`
```

Comment:
The tooltip texts here are a bit long and feel a bit "out of lore"; Maybe change to something like:
`Increases the energy consumed when reparing the hull by the given factor.`
This is a little bit less precise, but it seems to match the style of the `hull energy` text better and I think it should be fine for the average player.

Reply:
Well these could potentially be negative (after #4942), so do we want to use the term "increases"?

Reply:
You could add the text `(Decreases if the factor is negative)` to be complete.

Reply:
How do they look now?

Reply:
Looks good to me.